docs/testing.md

```diff
@@ -49,8 +49,8 @@ To install, you will have to allow installation from “unknown sources”.
 ### Log on to test app
 
 1. Install and open the MobilePay DK/FI MobilePay Test app
-2. Select "Log på (Eksisterende bruger)" / "Kirjaudu sovellukseen / "Log on (existing user)""
-3. Ensure that the environment selector is set to ”Integrator Sandbox (With Login)"
+2. Select "Log på (Eksisterende bruger)" / "Kirjaudu sovellukseen" / "Log on (existing user)"
+3. Ensure that the environment selector is set to "Sandbox (With Login)"
```

Comment:
Remove #3. It doesn't make any difference in terms of testing our feature and there'll be less confusion.